images/oadp-operator.yml

```
@@ -0,0 +1,36 @@
+cachito:
+  packages:
+    gomod:
+      - path: .
+content:
+  source:
+    dockerfile: konflux.Dockerfile
+    git:
+      branch:
+        target: oadp-1.5
+      url: git@github.com:openshift-priv/oadp-operator.git
+      web: https://github.com/openshift/oadp-operator
+distgit:
+  component: oadp-operator-container
+delivery:
+  delivery_repo_names:
+  - oadp/oadp-rhel9-operator
+for_payload: false
+for_release: false
+enabled_repos:
+- rhel-9-appstream-rpms
+- rhel-9-baseos-rpms
+from:
+  builder:
+    - stream: rhel-9-golang
+  member: openshift-enterprise-base-rhel9
+name: oadp/oadp-rhel9-operator
+owners:
+- rjohnson@redhat.com
+konflux:
+  network_mode: open
+update-csv:
+  bundle-dir: ''
```

OADP doesn't use a structure like metallb example.  It is just a /bundle/manifests/ rather than /bundle/{some-dir}/manifests/.  Would we then use `manifests/' here?
```suggestion
  bundle-dir: 'manifests/'
```